Workflow Docs: Add screenshots and workflow JSON download links (#20)
geocortex/dev-center: docs/workflow/sdk-net-getting-started-mobile.mdx

## SDK Samples

> The Geocortex Mobile SDK includes various [SDK Samples](../mobile/sdk-samples.mdx) for development, including ones like [this one](https://github.com/geocortex/vertigis-mobile-samples/tree/master/Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/Workflow/CustomActivity) which demonstrate custom workflow functionality.

Reviewer: ```suggestion
The Geocortex Mobile SDK includes various [SDK Samples](../mobile/sdk-samples.mdx) for development, including [a sample which demonstrate custom workflow functionality](https://github.com/geocortex/vertigis-mobile-samples/tree/master/Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/Workflow/CustomActivity) .
```
